nemo_skills/inference/eval/hilbench.py

```
@@ -0,0 +1,714 @@
+
+    def get_logs(self, timeout: int = 30) -> dict | None:
+        import urllib.request
+
+        try:
+            req = urllib.request.Request(
+                f"http://127.0.0.1:{self.port}/get_logs",
+                data=b"{}",
+                headers={"Content-Type": "application/json"},
+                method="POST",
+            )
+            with urllib.request.urlopen(req, timeout=timeout) as response:
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)